test/Feature/DeleteSpec.hs

```diff
@@ -27,15 +27,25 @@ spec =
           { matchStatus  = 200
           , matchHeaders = ["Content-Range" <:> "*/1"]
           }
+
+      it "ignores ?select= when return=minimal" $
+        request methodDelete "/items?id=eq.3&select=id" [("Prefer", "return=minimal")] ""
```

Comment:
Since now we default to the `return=minimal` behavior, I think these tests shouldn't contain the `Prefer: return=minimal`. Having the header confused me a little bit when reading the tests, I think the header should only be used when testing POST. Also, maybe the test should be called like `ignores ?select= when no return=representation`. Others could follow the same kind of naming.

What do you think?

Reply:
I tend to approach it the other way around. There were some tests (I think those with reduced privileges on columns), where a certain default setting for `return` was expected. This confused me! ;)

I think in general it would be best if all tests were specific about `return=representation` or `return=minimal` and would **never** assume a default. Because then those tests would all still work when the default should change at any point.

There should maybe be 1 test to actually test the default itself, to make sure that this doesn't change unintendedly.

Reply:
I see what you mean, that makes sense. Adding a test for the default would be great.